calculator/operations.py

```diff
@@ -1,19 +1,24 @@
 
 def add(*args):
-    # your implementation here
-    pass
-
+    return sum(args)
+    
 def subtract(*args):
-    # your implementation here
-    pass
+    return args[0] - sum(args[1:])
```

Comment:
Wow, awesome way to write this subtract function. We difficulty figuring out how to deal with the *args.